scripts/publish-root.gradle

```diff
@@ -14,12 +13,11 @@ if (secretPropsFile.exists()) {
     p.each { name, value -> ext[name] = value }
 } else {
     // Use system environment variables
-    ext["ossrhUsername"] = System.getenv('OSSRH_USERNAME')
-    ext["ossrhPassword"] = System.getenv('OSSRH_PASSWORD')
-    ext["sonatypeStagingProfileId"] = System.getenv('SONATYPE_STAGING_PROFILE_ID')
-    ext["signing.keyId"] = System.getenv('SIGNING_KEY_ID')
-    ext["signing.password"] = System.getenv('SIGNING_PASSWORD')
-    ext["signing.secretKeyRingFile"] = System.getenv('SIGNING_SECRET_KEY_RING_FILE')
+    ext["ossrhUsername"] = System.getenv("OSSRH_USERNAME")
+    ext["ossrhPassword"] = System.getenv("OSSRH_PASSWORD")
+    ext["sonatypeStagingProfileId"] = System.getenv("SONATYPE_STAGING_PROFILE_ID")
+    ext["signing.key"] = System.getenv("SIGNING_KEY")
+    ext["signing.password"] = System.getenv("SIGNING_PASSWORD")
```

Comment:
to me it looks like this env variable is not set in the action, is this correct ?

Reply:
Is it set here https://github.com/astarte-platform/astarte-device-sdk-java/blob/cc2677e439108d2a2a5842c241ec27e924ea185f/.github/workflows/sonatype-publish.yaml#L41?

Reply:
You are setting `OSSRH_PASSWORD` there, but the code looks for `SIGNING_PASSWORD` or am I missing something ?

Reply:
Ah ok i will add also `SIGNING_PASSWORD` to the system env